playbooks/roles/katello_provisioning/tasks/libvirt.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: 'install the virtualization hypervisor package group'
+  yum:
+    name: "@Virtualization Hypervisor"
+    state: present
+
+- name: 'install the virtualization tools package group'
+  yum:
+    name: "@Virtualization Tools"
+    state: present
+
+- name: 'disable libvirt authentication'
+  lineinfile:
+    dest: /etc/libvirt/libvirtd.conf
+    line: 'auth_unix_rw = "none"'
+
+- name: 'restart libvirt'
+  service: name=libvirtd state=restarted
```

Comment:
I'd been looking into ansible-lint and whether mixed case was something we should abide by stylistically or opt for expanded case everywhere (i.e. such as the rest of this yaml file)